spec/index.html

```diff
@@ -5462,29 +5462,33 @@ <h5>RDFterm-equal</h5>
                                                                                                                                                                                                              "name">term2</span></pre>
-            <p>Returns TRUE if <code>term1</code> and <code>term2</code> are the same RDF term as
-              defined in [[[RDF12-CONCEPTS]]] [[RDF12-CONCEPTS]]; produces a type error if the
-              arguments are both literal but are not the same RDF term <sup><a href=
-                                                                               "#func-RDFterm-equal-foot1" class="footnote">*</a></sup>; returns FALSE otherwise.
-              <code>term1</code> and <code>term2</code> are the same if any of the following is
+            <p>Returns TRUE if <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms as defined below;
+              produces a type error if the arguments are both literals with the same datatype that is unsupported by the implementation but are not the same RDF term <sup><a href="#func-RDFterm-equal-foot1" class="footnote">*</a></sup>;
+              returns FALSE otherwise.</p>
+            <p>
+              <code>term1</code> and <code>term2</code> are <em>equal</em> RDF terms if any of the following is
               true:</p>
             <ul>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are equivalent
+                <span class="name">term1</span> and <span class="name">term2</span> are equal
                 <span class="IRI type">IRIs</span> as defined in <a data-cite=
-                                                                    "RDF12-CONCEPTS#section-Graph-URIref">6.4 RDF URI References</a> of
+                                                                    "RDF12-CONCEPTS#dfn-iri-equality">IRI quality</a> of
                 [[RDF12-CONCEPTS]].
               </li>
               <li>
-                <span class="name">term1</span> and <span class="name">term2</span> are equivalent
+                <span class="name">term1</span> and <span class="name">term2</span> are equal
                 <span class="literal type">literals</span> as defined in <a data-cite=
-                                                                            "RDF12-CONCEPTS#section-Literal-Equality">6.5.1 Literal Equality</a> of
+                                                                            "RDF12-CONCEPTS#dfn-term-equal">Literal term equality</a> of
                 [[RDF12-CONCEPTS]].
               </li>
+              <li>
+                <span class="name">term1</span> and <span class="name">term2</span> are
+                <span class="literal type">literals</span> with equivalent values according to their datatypes.
+              </li>
```

Comment:
I think that this new bullet point to you have added to the definition here is problematic. First of all, it is not absolutely clear what it means for literals to have "equivalent values according to their datatypes."

Here is a proposal to improve this definition:
```suggestion
              <li>
                <span class="name">term1</span> is a <span class="literal type">literal</span> and
                <span class="name">term2</span> is a <span class="literal type">literal</span>
                such that the <a data-cite="RDF12-CONCEPTS#dfn-datatype-iri">datatype IRI</a>
                of these two literals is the same, this IRI is in the set of
                <a data-cite="RDF12-CONCEPTS#dfn-recognized-datatype-iri">recognized datatype IRIs</a>,
                and the <a data-cite="RDF12-CONCEPTS#dfn-literal-value">literal value</a>
                associated with both literals is the same.
              </li>
```